.build/website/src/pages/index.module.css

```diff
@@ -3,16 +3,15 @@
  * and scoped locally.
  */
 
-.heroBanner {
+ .heroBanner {
   padding: 4rem 0;
   text-align: center;
   position: relative;
   overflow: hidden;
   background-repeat: no-repeat;
   background-position: top right;
-  /*sample for image*/
-  /*background-image: -webkit-image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
-  /*background-image: image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
+  background-image: -webkit-image-set(url(https://everscale.network/images/1x.png) 1x, url(https://everscale.network/images/2x.png) 2x);
```

Comment:
https://everscale-incubator.github.io/docs/PR-308/
![image](https://user-images.githubusercontent.com/761285/209577317-d4b937e6-5a2b-4191-a1ce-d0e09c9e1db4.png)

Reply:
Can you suggest how to fix this?

Reply:
Can you suggest how to fix this?

Reply:
> https://everscale-incubator.github.io/docs/PR-308/
> ![image](https://user-images.githubusercontent.com/761285/209577317-d4b937e6-5a2b-4191-a1ce-d0e09c9e1db4.png)
> 

Can you suggest how to fix this?

Reply:
@babowsk1 https://hexlet.io/programs/frontend or https://djinni.co/developers/ or  https://github.com/everscale-org/bounties/issues/new/choose

Reply:
@babowsk1 or postpone the clearance decision and change the focus to higher priority documentation issues